server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/QueueSummaries.java

```diff
@@ -100,6 +100,10 @@ public String toString() {
     }
   }
 
+  synchronized boolean isCompactionsQueued(String queue) {
+    return QUEUES.containsKey(queue) && QUEUES.get(queue) != null && !QUEUES.get(queue).isEmpty();
```

Comment:
It makes the code longer, but the following does less map lookups.

```suggestion
    var q = QUEUES.get(queue);
    if(q == null) {
      return false;
    }
    return !q.isEmpty();
```

Reply:
Implemented in 6cf5f76